devel/ansible/roles/dev/files/sar.py

```diff
@@ -0,0 +1,95 @@
+import click
+import json
```

Comment:
In Python there is a tradition to separate imports in the following manner:
1. imports from the stdlib
2. blank line
3. imports from third-party libs
4. blank line
5. imports from the project

In this case it would become:
```
import json

import click
from python_freeipa import ...
```